### DISCUSSION TITLE
All in one

### DISCUSSION THREAD

**Discussion by ramkdlr** [BennyThadikaran/eod2#56] [General]
Hi gone through your EOD2 wonderful initiative could you merge the delivery and EOD OHLCV to form a single CSV?.

**Comment by BennyThadikaran** [comment]
@ramkdlr 
Hi, I'm glad you bought this up. :+1: 

It has many benefits:
- It will reduce the number of files and save repo space.
- Some overall code reduction
- Users may find it easier to work with one file

I had considered this in the past but dismissed it. My concern was whether NSE would discontinue or change the structure of its delivery reports. It is not a concern anymore.

I'll start working on EOD2 next month and will create a github issue when i start work on this.
Meanwhile, I look into this and post my thoughts here.

**Reply by ramkdlr** [reply]
Great will wait for the update.

**Comment by BennyThadikaran** [comment]
I just had a quick look at this today. Below is how the columns would look after the merge.

`Open, High, Low, Close, Volume, TOTAL_TRADES, QTY_PER_TRADE, DLV_QTY`

`TTL_TRD_QNTY` is same as `Volume`.
`NO_OF_TRADES` renamed to `TOTAL_TRADES`
`DELIV_QTY` renamed to `DLV_QTY`

I'll start work on this next week.

**Reply by ramkdlr** [reply]
ok great.

**Comment by BennyThadikaran** [comment]
@ramkdlr Raised issue #62 . Work is almost done. But need to make changes in my other scripts and some final testing before i merge. So most likely by tomorrow evening.

**Comment by BennyThadikaran** [comment]
@ramkdlr I have completed this feature. See all the changes #69

I will post a Changelog and Update instructions on Discussions tomorrow.
